run/pubsub/go.mod

```diff
@@ -1,3 +1,3 @@
 module github.com/GoogleCloudPlatform/golang-samples/run/pubsub
 
-go 1.12
+go 1.13
```

Comment:
This signifies the Go version this module is compatible with. Are we sure we want to bump it?

Reply:
Undone per suggestion that this not be bumped.